config/nf_modules.config

```diff
@@ -192,59 +204,71 @@ process {
                         if ( !filename.endsWith(".out") ) {
                             return null;
                         }
-                        def output_file = file(filename);
+                        def output_file = new File(filename);
                         def genome_id = fasta.baseName;
                         if ( output_file.name.contains("_rRNAs") || output_file.name.contains("_tRNA_20aa") ) {
-                            return "${genome_id}/${genome_id}_out-results/${output_file.name}";
+                            return "${genome_id}/RNA/${genome_id}_out/${output_file.name}";
```

Comment:
I'm sure MAG_Uploader will not work with extra RNA folder level. Old script https://github.com/EBI-Metagenomics/assembly-pipeline/blob/develop/pipeline_lib/path_finder.py#L108

Reply:
OK, we may need to adjust the MAG Uploader then. At this point I think we need to define a very clear structure for the outputs.

Also, I feel we should go with all lowercase folder and file names